### ISSUE TITLE
Cannot access web interface

### ISSUE THREAD

**Issue by impy101** [ccrisan/motioneyeos#2957]
I have installed Docker and Armbian Buster on my BPI M2 zero and I have succesfully (I think) added Motioneye to my containers but I cannot open the web interface with <board IP>:8765

Have I missed something?

### Preliminary Docs

I confirm that I have read the [CONTRIBUTING](https://github.com/ccrisan/motioneyeos/blob/master/.github/CONTRIBUTING.md) guide before opening this issue.

I confirm that I have read the [FAQ](https://github.com/ccrisan/motioneyeos/wiki/FAQ) before opening this issue.

### motionEyeOS Version

I am running motionEyeOS version: ccrisan/motioneye:master-amd64

### Board Model

I am using the following board/model: BPI m2 zero

### Network Connection

My motionEyeOS unit is connected to the network via: Wifi

**Comment by starbasessd** [comment]
Yes. You are running on an ARM machine, not a AMD64. You are running the wrong version. Follow these instructions for the ARM version:
docker pull ccrisan/motioneye:master-armhf
